test/test_openmp.py

```diff
@@ -0,0 +1,87 @@
+import collections
+import sys
+import time
+import unittest
+
+import torch
+from torch.testing._internal.common_utils import TestCase, run_tests 
+
+try:
+    import psutil
+    HAS_PSUTIL = True
+except ImportError:
+    HAS_PSUTIL = False
+
+device = torch.device('cpu')
+
+
+# Use the C function rather than trying to set the environment variable
+# OMP_NUM_THREADS since on some OSs os.environ['OMP_NUM_THREADS'] only takes
+# effect on new processes
+
+class Network(torch.nn.Module):
+    maxp1 = torch.nn.MaxPool2d(1, 1)
+
+    def forward(self, x):
+        return self.maxp1(x)
+
+
+@unittest.skipIf(
+    not HAS_PSUTIL,
+    "Requires psutil to run")
+class TestOpemMP_ParralelFor(TestCase):
+    batch = 20
+    channels = 1
+    side_dim = 80
+    x = torch.randn([batch, channels, side_dim, side_dim], device=device)
+    model = Network()
+    ncores = min(5, psutil.cpu_count(logical=False))
+
+    def func(self, runs):
+        p = psutil.Process()
+        # warm up for 5 runs, then things should be stable for the last 5
+        last_rss = collections.deque(maxlen=5)
+        for n in range(10):
+            for i in range(runs):
+                self.model(self.x)
+            last_rss.append(p.memory_info().rss)
+        return last_rss
+
+    def func_rss(self, runs):
+        last_rss = self.func(runs)
+        tf = all([dd == last_rss[-1] for dd in last_rss])
+        self.assertTrue(tf, msg='memory did not stabalize')
+
+    @unittest.skipIf(sys.platform == 'win32' or ncores < 5,
```

Comment:
This doesn't run. `sys` is not imported and `ncores` doesn't exist in the global scope.

Reply:
fixed